app/models/rules/basic.rb

```diff
@@ -1,4 +1,4 @@
-class Rules::Basic < Struct.new(:auction)
+class Rules::Basic < Rules::BaseRules
```

Comment:
Maybe we should call the default auction something else since we have Base Rules now. Default? Reverse?